elements/pfe-styles/src/pfe-typography.scss

```diff
@@ -0,0 +1,65 @@
+@import "../../pfe-sass/pfe-sass";
+
+// HEADINGS
+.pfe-heading-3xl {
+    @extend %pfe-heading--3xl;
+}
+.pfe-heading-3xl {
+    @extend %pfe-heading--3xl;
+}
+.pfe-heading-2xl {
+    @extend %pfe-heading--2xl;
+}
+.pfe-heading-xl {
+    @extend %pfe-heading--xl;
+}
+h1,
```

Comment:
Does it make sense to put the styles for the default heading tags in a different file than the classes? For example, I may want to make use of the new PFE classes which are opt-in, but I may not want the h2 styles to interfere with what I already have going on with my website.
@castastrophe @LyndseyR

Reply:
I think allowing people to opt-in as much as possible is ideal. I'm going to split these out for now, we can always re-combine them if needed.

Reply:
The resulting fils are now: 
```
pfe-layouts.css (pre-existing)
pfe-context.css (pre-existing)
pfe-base.css  (pre-existing)
---
pfe-text-elements.css
pfe-typography-mobile.css
pfe-typography-deprecated.css  (will be deleted eventually)
pfe-typography-classes.css
```

Reply:
continuing the convo with mahself, I'm thinking we should combine pfe-base & pfe-text-elements since base already contains those things